### PR TITLE
News pagination

### PR DESCRIPTION
**Pull request by yaroslavbilokin**

### CODE REVIEW

File: src/configs/index.js [hunk 1]
```diff
@@ -58,5 +58,10 @@ export const config = {
     email:
       '^([\\w-]+(?:\\.[\\w-]+)*)@((?:[\\w-]+\\.)*\\w[\\w-]{0,66})\\.([a-z]{2,6}(?:\\.[a-z]{2})?)$',
     password: '^(?!.* )(?=.*[0-9])(?=.*[A-Z]).{8,30}$'
+  },
+  paginationPayload: {
+    skip: 0,
+    limit: 5,
+    productsPerPage: 5
```
Comment (VolodymyrTrach) [inline src/configs/index.js:65]: We have 3 news per row. It will be nicer if it will be 3/6/9. I think 6 will be alright

Comment (yaroslavbilokin) [reply src/configs/index.js]: Changed!

File: src/redux/news/news.reducer.js [hunk 2]
```diff
@@ -2,14 +2,20 @@ import {
...
+  SET_PAGES_COUNT
 } from './news.types';
 
 const initialState = {
   list: [],
   newsArticle: null,
   newsLoading: false,
-  newsError: null
+  newsError: null,
+  currentPage: 0,
```
Comment (Alan18081) [inline src/redux/news/news.reducer.js:16]: Please, put all of the pagination fields inside its own `pagination` object

Comment (yaroslavbilokin) [reply src/redux/news/news.reducer.js]: Changed!

**Pull request merged**